src/AcceptanceTests/AcceptanceTests.csproj

```diff
@@ -13,7 +13,9 @@
     <PackageReference Include="NServiceBus.Azure.Transports.WindowsAzureStorageQueues" Version="8.0.0-*" />
     <PackageReference Include="NServiceBus.Newtonsoft.Json" Version="2.0.0-*" />
     <PackageReference Include="NServiceBus" Version="7.0.0-*" />
-    <PackageReference Include="NUnit" Version="3.7.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.*" />
+    <PackageReference Include="NUnit" Version="3.7.*" />
```

Comment:
Not `3.*`?

Reply:
i was trying to debug a build server issue. but yes is should be 3.*